### PR TITLE
added developer documentation

### PR DESCRIPTION
**Pull request by wadeking98**
Signed-off-by: wadeking98 <wkingnumber2@gmail.com>

### CODE REVIEW

File: DEVELOPER/BC Wallet Showcase.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+To modify the deployment config, make sure you're in the project's openshift directory. Then run the following command, swap out occurrences of `dev` for the environment you're trying to update.
+```bash
+genDepls.sh -n a99fd4-dev -e dev -u
+```
+There may be one or two errors/warnings, just press enter to get through them.
+
+#### Pipelines
+There are three pipelines for the showcase: `bc-wallet-demo-pipeline`, `bc-wallet-demo-deploy-to-test-pipeline`, and `bc-wallet-demo-deploy-to-prod-pipeline`.  `bc-wallet-demo-pipeline` deploys to dev and gets triggered whenever someone pushes to the github repo. This pipeline runs the build, creates a new image stream, and tags it as dev. If this pipeline gets stuck in the `new` state and never builds, it's probably because there's too many old builds in the environment. Delete some of the old builds and it should work again.
+
+The other two pipelines are triggered manually, the test pipeline points the test image stream tags to dev, and the prod pipeline points the prod image stream tags to test. This means to deploy changes to prod, the test pipeline must run first.
```
Comment (bryce-mcmath) [inline DEVELOPER/BC Wallet Showcase.md:332]: Sorry if this was covered but how do you actually manually trigger those two pipelines?

**Review by bryce-mcmath — COMMENTED** [review]
Just one question so far

File: DEVELOPER/BC Wallet Showcase.md (hunk 1 above)
Comment (wadeking98) [reply DEVELOPER/BC Wallet Showcase.md:332]: You can trigger them manually by going to the openshift buildconfig section in a99fd4-tools and clicking selecting `build` from the menu under the elipses

Comment (wadeking98) [reply DEVELOPER/BC Wallet Showcase.md:332]: I've updated the docs to include more info on it

**Pull request merged**